modules/nw-configuring-sriov-rdma-cni.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-rdma-cni.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-configuring-sriov-rdma-cni_{context}"]
+= Configuring SR-IOV RDMA CNI
+
+You can configure an RDMA CNI on SR-IOV.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create an `SriovNetworkPoolConfig` CR and save it as `sriov-nw-pool.yaml`, as in the following example:
++
+.Example `SriovNetworkPoolConfig` CR
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: worker
+  namespace: openshift-sriov-network-operator
+spec:
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/workercnf: ""
```

Comment:
please use worker here the same way we did in https://docs.openshift.com/container-platform/4.17/networking/hardware_networks/configuring-sriov-device.html#configure-sr-iov-operator-parallel-nodes_configuring-sriov-device

Reply:
also it's important to add `maxUnavailable: 1` and maybe link to the parallel draining